apps/system/js/sync_manager.js

```diff
@@ -746,8 +762,15 @@
 
     user: {
       set: function(user) {
-        asyncStorage.setItem(SYNC_USER, user, () => {
-          this.store.set(SYNC_USER, user);
+        // We need to check if there's an user logged in during the enabled
```

Comment:
Nit: an user -> a user. Phonetically, "yoozer" :) starts with a consonant, and that's what decides "a"/"an"